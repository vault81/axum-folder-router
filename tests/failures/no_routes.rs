use axum_folder_router::folder_router;

#[derive(Clone)]
struct AppState;

#[folder_router("../../../../tests/failures/no_routes", AppState)]
struct MyFolderRouter();

fn main() {}
