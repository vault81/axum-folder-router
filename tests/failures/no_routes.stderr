error: No routes defined in your route.rs's !
       Ensure that at least one `pub async fn` named after an HTTP verb is defined. (e.g. get, post, put, delete)
 --> tests/failures/no_routes.rs:6:1
  |
6 | #[folder_router("../../../../tests/failures/no_routes", AppState)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `folder_router` (in Nightly builds, run with -Z macro-backtrace for more info)
